macOS-HS-libvirt.xml

```diff
@@ -50,9 +46,8 @@
   </os>
   <features>
     <acpi/>
-<!-- Note: On Fedora 27 w/ an i7-5820K CPU and repository QEMU version as of 27-Jan-2018, the guest experienced severe clock issues. If you encounter these issues, try changing this to <hidden state ='off'/> -->
     <kvm>
-      <hidden state='on'/>
+      <hidden state='off'/>
```

Comment:
Is this change really required or can we keep the former value?

Reply:
This change is required for the tsc clock feature